tests/unit/sample_apps/test_reviewer_app.py

```
@@ -0,0 +1,43 @@
+SPEC = importlib.util.spec_from_file_location("reviewer_app", APP_PATH)
+reviewer_app = importlib.util.module_from_spec(SPEC)
+assert SPEC.loader is not None
+SPEC.loader.exec_module(reviewer_app)
+ReviewerApp = reviewer_app.ReviewerApp
+
+
+class TestReviewerAppConfig(unittest.TestCase):
+    def test_conf_mode_is_preserved_without_env_override(self):
+        app = ReviewerApp.__new__(ReviewerApp)
+        with patch.dict(os.environ, {}, clear=False):
+            config = app._load_review_config("/tmp/studies", {"mode": "standalone"})
```

Ruff (0.15.21)

[error] Probable insecure usage of temporary file or directory: "/tmp/studies"

(S108)